index.html

```diff
@@ -203,21 +200,21 @@ <h2 class="newsletter-title">
       <h2 class="visually-hidden">Мы в соц.сетях</h2>
       <ul class="footer-social-list">
         <li class="footer-social-item">
-          <a class="button" href="#">
+          <a class="social-link" href="#">
```

Comment:
Стоит соблюдать пространство имен:
```suggestion
          <a class="footer-social-link" href="#">
```